ui/pi-web-agent-app/src/app/live-info/live-info.component.ts

```diff
@@ -8,24 +8,24 @@ import { SystemInfo, SystemInfoService } from '../system-info.service';
 })
 export class LiveInfoComponent implements OnInit {
 
-  constructor(private system_info_service: SystemInfoService) { }
+  constructor(private systemInfoService: SystemInfoService) { }
 
-  system_info: SystemInfo = {
-    Temperature: "",
-    Kernel: "",
+  systemInfo: SystemInfo = {
+    Temperature: '',
```

Comment:
why is it replacing double quotes?

Reply:
tha's from our lint configuration I didn't  do it manually